contrib/debian/rules

```diff
@@ -12,10 +12,16 @@ override_dh_auto_clean:
 	if [ -f Makefile ]; then $(MAKE) distclean; fi
 	rm -rf Makefile.in aclocal.m4 configure src/Makefile.in src/bitcoin-config.h.in src/build-aux src/qt/Makefile.in src/qt/test/Makefile.in src/test/Makefile.in
 
+ifeq ($(shell dpkg-vendor --derives-from Ubuntu && echo yes),yes)
```

Comment:
Maybe lets wait until 10.17 before we lock in using qt5 in 17.10 in case they slip to the next release or so?